utils/installer/install.ps1

```diff
@@ -0,0 +1,136 @@
+		ForEach-Object {$_ -Replace 'os.getenv "HOME"', 'os.getenv "HOMEDRIVE" .. os.getenv "HOMEPATH"'} |
+			Set-Content -Path $config
+}
+
+function cloneConfig($arg) {
+	Write-Output "Cloning LunarVim configuration"
+	mkdirOnDemand("$XDG_DATA_HOME\lunarvim")
+	
+	if ($arg.Contains("--testing")) {
+		Copy-Item -Recurse "$pwd" "$XDG_DATA_HOME\lunarvim\lvim"
+	} else {
+		Write-Output "lvbranch=$LVBRANCH"
+		git clone --branch "$LVBRANCH" "https://github.com/LunarVim/LunarVim.git" "$XDG_DATA_HOME\lunarvim\lvim"
+	}
+
+	mkdirOnDemand("$XDG_CONFIG_HOME\lvim")
+	
+	if (($env:Path.Contains("$XDG_DATA_HOME\lunarvim\lvim\utils\bin\")) -eq $false) {
+		$env:Path += ";$XDG_DATA_HOME\lunarvim\lvim\utils\bin\"
+		[Environment]::SetEnvironmentVariable("Path", $env:Path, [System.EnvironmentVariableTarget]::User)
+	}
+	
+	# Hack because in rolling branch lv-config.lua is renamed to config.lua
+	if ($LVBRANCH -eq "rolling") {
+		Copy-Item "$XDG_DATA_HOME\lunarvim\lvim\utils\installer\config.example-no-ts.lua" "$XDG_CONFIG_HOME\lvim\config.lua"
+	} else {
+		Copy-Item "$XDG_DATA_HOME\lunarvim\lvim\utils\installer\lv-config.example-no-ts.lua" "$XDG_CONFIG_HOME\lvim\lv-config.lua"
+	}
+
+	modifyHomeVar("$XDG_DATA_HOME\lunarvim\lvim\lua\default-config.lua")
+	modifyHomeVar("$XDG_DATA_HOME\lunarvim\lvim\init.lua")
+
+	nvim -u $XDG_DATA_HOME\lunarvim\lvim\init.lua --cmd "set runtimepath+=$XDG_DATA_HOME\lunarvim\lvim" --headless +'autocmd User PackerComplete sleep 100m | qall' +PackerInstall
+	nvim -u $XDG_DATA_HOME\lunarvim\lvim\init.lua --cmd "set runtimepath+=$XDG_DATA_HOME\lunarvim\lvim" --headless +'autocmd User PackerComplete sleep 100m | qall' +PackerSync
+
+	Write-Output ""
+	Write-Output "Compile Complete"
+	Write-Output ""
+
+	if (Test-Path "$XDG_DATA_HOME\lunarvim\lvim\init.lua" -PathType Any) {
+		Write-Output "lv-config already present"
+	} else {
+		if($LVBRANCH -eq "rolling") {
+			Copy-Item "$XDG_DATA_HOME\lunarvim\lvim\utils\installer\config.example.lua" "$XDG_CONFIG_HOME\lvim\config.lua"
+		} else {
+			Copy-Item "$XDG_DATA_HOME\lunarvim\lvim\utils\installer\lv-config.example.lua" "$XDG_CONFIG_HOME\lvim\lv-config.lua"
+		}
+	}
+}
+
+function rewriteLvimForWindows {
+	Set-Content "$XDG_DATA_HOME\lunarvim\lvim\utils\bin\lvim.ps1" "nvim -u `"$XDG_DATA_HOME\lunarvim\lvim\init.lua`" --cmd `"set runtimepath+=$XDG_DATA_HOME\lunarvim\lvim`" `"$args`""
+}
+
+# Welcome
+Write-Output 'Installing LunarVim'
+if ($args.Contains("--override")) {
+    Write-Output '!!Warning!! -> Removing all lunarvim related config because of the --overwrite flag'
+	Remove-Item -Recurse -Force "$XDG_DATA_HOME\lunarvim" -ErrorAction Ignore
+	Remove-Item -Recurse -Force "$XDG_CACHE_HOME\nvim" -ErrorAction Ignore
+	Remove-Item -Recurse -Force "$XDG_CONFIG_HOME\lvim" -ErrorAction Ignore
+}
+
+# move old lvim directory if it exists
+if (Test-Path -Path "$XDG_DATA_HOME\lunarvim") { moveOldLvim }
+
+# install pip
+verifyInstalled("pip3.exe")
+Write-Output "pip installed, moving on..."
```

Comment:
We can use [ensurepip](https://docs.python.org/3/library/ensurepip.html) here and only bail out if that command failed.

```pwsh
python -m ensurepip --upgrade
```